src/components/Checkbox/Checkbox.stories.js

```diff
@@ -10,3 +10,6 @@ export function Default() {
 export function checked() {
 	return <StyledCheckbox task={{ done: false }} />;
 }
+export function old() {
+	return <checkbox checked />;
+}
```

Comment:
Is it there in order to get a comparison between the default styling of a checkbox and your newly created one?